packages/design-system-react/src/components/button-link/ButtonLink.types.ts

```diff
@@ -0,0 +1,47 @@
+import type { ButtonBaseProps } from '../button-base';
+import { ButtonBaseSize } from '../button-base';
+
+export enum ButtonLinkSize {
+  Sm = ButtonBaseSize.Sm,
+  Md = ButtonBaseSize.Md,
+  Lg = ButtonBaseSize.Lg,
+  /**
+   * Inherits font size from parent, removes height/padding, displays inline
```

Comment:
Updated from `Inherit size from parent` to `Inherits font size from parent`

Reply:
that's not what this size does though? It displays inline with no bounding box size and padding, but the size will instead be content-based